lib/components/home/artwork_rails/artwork_rail_header.js

```diff
@@ -0,0 +1,100 @@
+/* @flow */
+'use strict'
+
+import Relay from 'react-relay'
+import React from 'react'
+import { StyleSheet, Dimensions, View, Text } from 'react-native'
+
+import SerifText from '../../text/serif'
+import colors from '../../../../data/colors'
+import Button from '../../buttons/inverted_button'
+
+const isPad = Dimensions.get('window').width > 700
+
+class ArtworkRailHeader extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        { this.props.rail.context && this.followAnnotation() }
+        <SerifText numberOfLines={2} style={styles.title}>{ this.props.rail.title }</SerifText>
+        { this.props.rail.context && this.actionButton() }
+      </View>
+    )
+  }
+
+  followAnnotation() {
+    // string comparing feels super weird but making a javascript 'enum' might be worse?
```

Comment:
an alternative would be to `const showSubtitle = this.props.rail.context.__typename === 'HomePageModuleContextRelatedArtist'` it out, and force some specify, but I'm not sure you gain too much